alternator/executor.hh

```diff
@@ -78,6 +93,18 @@ public:
     future<> create_keyspace(std::string_view keyspace_name);
 
     static tracing::trace_state_ptr maybe_trace_query(client_state& client_state, sstring_view op, sstring_view query);
+
+    static sstring table_name(const schema&);
```

Comment:
The neat thing about class-private functions it that they are class-private. I.e. if the intent is to use them for the purposes of the various calls in executor, we know no one will use them elsewhere. Of course this broke here, because of all the op-classes also using the free functions, and those not being class local. 

If you want me to move stuff to shared globals instead, then I ask for a header name (executor-internal.hh?).

Reply:
Honestly, I am 100% ok with moving helpers into a module-private header + cc, but I suggest making this a later refactoring.